test/framework/kubetest/run.go

```diff
@@ -28,6 +28,7 @@ import (
 	"strings"
 
 	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
```

Comment:
I'll switch this import.
Need to do a little linter that stops us accidentally importing this instead of the sigs one.